servers/express.js

```diff
@@ -2,32 +2,39 @@
 
 const fs = require('fs');
 const path = require('path');
+const express = require('express');
 
 const Skeleton = require('./skeleton');
 const watcher = require('../file-watcher.js');
 
 // the app
-const express = require('express');
 const app = express();
 
-const pathToLogFile = path.join(__dirname, '..', '..', 'node-route-metrics', 'route-metrics.log');
+const pathToLogFile = (process.argv.length > 2) ? process.argv[2] : 'router-metrics.log';
 const htmlTemplate = fs.readFileSync(path.join(__dirname, 'pages/templatized.html'), 'utf8');
 
 app.get('/eventloop', function(req, res) {
   const chartOpt = makeChartOptions({title: 'Eventloop lag percentiles (ms)', subtitle: '@contrast/route-metrics'});
   const datarows = makeDataRows(eventloopDataRows);
-  const columns  = makeColumnNames();
-
-  const html = populateTemplate(htmlTemplate, chartOpt, columns, datarows);
+  
+  const html = populateTemplate(htmlTemplate, chartOpt, makeEventloopColumnNames(), datarows);
```

Comment:
probably want to make event loop column names the same form as data rows, e.g.,

`const columnNames = makeEventloopColumnNames();` and then use that in the call to `populateTemplate()`.